optaweb-vehicle-routing-docs/src/main/asciidoc/modules/con-vrp.adoc

```diff
@@ -0,0 +1,11 @@
+[id="con-vrp_{context}"]
```

Comment:
Can we use the following custom ID syntax?
```
[#con-vrp_{context}]
```
It's simpler.

Reply:
No :-) That syntax is required by the tooling team. Sorry. And we do need the _context placeholder.